HashSet.py

```diff
@@ -0,0 +1,42 @@
+class MyHashSet:
```

Comment:
Try the solution taught in class, it'll improve your time as well as space efficiently and also, try to write functions if you feel some part of the code is repeated. In this case, finding the index (key%1000)